src/assets/js/protocol/protocol-newsletter.js

```diff
@@ -0,0 +1,47 @@
+(function() {
+    'use strict';
+
+    // !! This file assumes only one signup form per page !!
+
+    // Expand email form on input focus or submit if details aren't visible
+    function initEmailForm() {
+        var newsletterForm = document.getElementById('newsletter-form');
+        var submitButton = document.getElementById('newsletter-submit');
+        var formDetails = document.getElementById('newsletter-details');
+        var formControls = document.querySelectorAll('.mzp-c-newsletter-form input, .mzp-c-newsletter-form select');
+        var formExpanded;
+
+        if (window.getComputedStyle(formDetails).display === 'none') {
```

Comment:
If this is condition falsy then the value for `formExpanded` will be `null` instead of `true`. Maybe something like this could work?

```javascript
var formExpanded = window.getComputedStyle(formDetails).display === 'none' ? false : true;
```